bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -90,90 +86,58 @@ <h3 id="explore">{{_('Explore the site') }}</h3>
       <div class="article-content">
         <h2>{{ _('Unleash the future') }}</h2>
         <div class="intro">
-
           <p>
-            {{ _('In our first year, we’ve brought openness, accessibility and opportunity to 15 countries with 4 hardware partners, 7 operators and 16 launches.') }}
-          {% trans url1='class="nav" href="#os"'|safe, url2='class="nav" href="#marketplace"'|safe, url3='class="nav" href="#android"'|safe %}
-            See what’s next for mobile with <a {{url1}}>Firefox OS</a>, <a {{url2}}>Firefox Marketplace</a>
-            and <a {{url3}}>Firefox for Android</a>.
-          {% endtrans %}
+            {% if l10n_has_tag('mwc_2015') %}
+              {{ _('In our first year, we’ve brought openness, accessibility and opportunity to 15 countries with 4 hardware partners, 7 operators and 16 launches.') }}
+            {% else %}
+              {{ _('Firefox OS has brought openness, accessibility and opportunity to 30 countries with more than 50 launches and 14 global operators — and that momentum continues in 2015.') }}
+            {% endif %}
+            {% trans url1='class="nav" href="#os"'|safe, url2='class="nav" href="#marketplace"'|safe, url3='class="nav" href="#android"'|safe %}
+              See what’s next for mobile with <a {{url1}}>Firefox OS</a>, <a {{url2}}>Firefox Marketplace</a>
+              and <a {{url3}}>Firefox for Android</a>.
+            {% endtrans %}
           </p>
 
-          <p class="partner-button"><a href="https://mobilepartners.mozilla.org/" class="button">{{ _('Partner with us') }}</a></p>
+          <p class="partner-button"><a href="https://mobilepartners.mozilla.org/" class="button-flat">
+          {% if l10n_has_tag('mwc_2015') %}
+            {{ _('Be a partner') }}
+          {% else %}
+            {{ _('Partner with us') }}
+          {% endif %}
+          </a></p>
 
           <div id="overview-news-links">
             <div id="overview-news-wrapper">
-              <img src="{{ static('img/firefox/partners/mwc-2014-banner.png') }}" alt="{{ _('Mobile World Congress 2014') }}">
+              <img src="{{ static('img/firefox/partners/mwc-2015-banner.png') }}" alt="{{ _('Mobile World Congress 2015') }}">
               <div id="overview-news">
+              {% set feed_links = firefox_os_feed_links(LANG) %}
+              {% if feed_links %}
+                {% set feed_link = feed_links[0] %}
```

Comment:
Not really a much **better** way, but here you could make it a bit more readable by doing:

```jinja
{% set feed_url, feed_title = feed_links[0] %}
```

For some reason the helper is returning tuples instead of model objects.